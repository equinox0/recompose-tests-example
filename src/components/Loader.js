import React from "react";

const Loader = () => <div>Loading data...</div>;

export { Loader };
export default Loader;
