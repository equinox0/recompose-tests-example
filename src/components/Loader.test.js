import { shallow } from "enzyme";
import React from "react";
import { Loader } from "./Loader";

describe("<Loader/>", () => {
    describe("component", () => {
        it("should render loader", () => {
            const wrapper = shallow(<Loader />);
            expect(wrapper).toMatchSnapshot();
        });
    });
});
