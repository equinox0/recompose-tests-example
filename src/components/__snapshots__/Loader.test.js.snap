// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> component should render loader 1`] = `
<div>
  Loading data...
</div>
`;
